pkg/httpclient/httpclient.go

```diff
@@ -0,0 +1,30 @@
+package httpclient
+
+import (
+	"fmt"
+	"net/http"
+)
+
+var Version string
+
+func Get(url string) (*http.Response, error) {
```

Comment:
This absolutely needs a doc comment describing that it adds the proper user-agent and returns non-200 status as an error.